tajo-common/src/main/java/org/apache/tajo/conf/TajoConf.java

```diff
@@ -203,6 +203,7 @@ public static int setDateOrder(int dateOrder) {
 
     // Query Configuration
     QUERY_SESSION_TIMEOUT("tajo.query.session.timeout-sec", 60, Validators.min("0")),
+    QUERY_SESSION_CACHE_SIZE("tajo.query.session.cache-size", 1000000, Validators.min("1000000")),
```

Comment:
I have a couple of comments on this line.

* The cache size seems to be specified in bytes. This will be difficult for humans to specify the exact cache size. How about using the size unit of KB? 
* The session variable name looks too general even though this cache is only for parsed queries. It would be better if users can figure out what will be configured from its name. Also, the size unit should be included in the name like ```tajo.task.size-mb```.
* The minimum cache size is 1 GB. Do you have any reasons?
* In addition to the size configuration, it would be great if users can turn off/on this cache feature. This is because the cache may be useless in some workloads such as ad-hoc analysis.